packages/select/theme/lumo/vaadin-select-styles.js

```diff
@@ -67,6 +68,12 @@ registerStyles(
       box-shadow: none;
     }
 
+    ::slotted(:not([slot])) {
+      min-height: var(--lumo-button-size);
+      padding-top: var(--_lumo-selected-item-padding);
+      padding-bottom: var(--_lumo-selected-item-padding);
```

Comment:
Aligned with the styles used by Vaadin 21:

https://github.com/vaadin/web-components/blob/824e52a9aea964e57cc2937c097a375097c24a05/packages/vaadin-select/theme/lumo/vaadin-select-styles.js#L26-L29